parser.c

```diff
@@ -478,16 +479,43 @@ static struct kanshi_profile *parse_profile(struct kanshi_parser *parser) {
 	}
 }
 
-static struct kanshi_config *_parse_config(struct kanshi_parser *parser) {
-	struct kanshi_config *config = calloc(1, sizeof(*config));
-	wl_list_init(&config->profiles);
+static int parse_config_file(const char *path, struct kanshi_config *config);
+
+static int parse_include_command(struct kanshi_parser *parser, struct kanshi_config *config) {
```

Comment:
I'd prefer to use `bool` as the return type instead of `int`, as done in the rest of the file.